raiden/transfer/state.py

```diff
@@ -539,7 +539,7 @@ class ChainState(State):
     identifiers_to_tokennetworkregistries: Dict[
         TokenNetworkRegistryAddress, TokenNetworkRegistryState
     ] = field(repr=False, default_factory=dict)
-    nodeaddresses_to_networkstates: Dict[Address, NetworkState] = field(
+    nodeaddress_to_networkstate: Dict[Address, NetworkState] = field(
```

Comment:
Does this get repopulated on every start? (If so, why does it even exist in the state).
If not this is backwards-incompatible.

Reply:
You're right! Let's do this at a point where we break compatibility for some other reason.